Formula/ec2instanceconnect.rb

```diff
@@ -0,0 +1,74 @@
+class Ec2instanceconnect < Formula
+  include Language::Python::Virtualenv
+
+  desc "All-in-one client for EC2 Instance Connect"
+  homepage "https://github.com/aws/aws-ec2-instance-connect-cli"
+  url "https://files.pythonhosted.org/packages/0c/7b/2dbd14b2caf0d834f781ef982bb4b22884245c44af0ebb6ccc5a7019fd3a/ec2instanceconnectcli-1.0.0.tar.gz"
+  sha256 "dc818c78d51a475a97d502db0221962e3c42af66900a63d7da69d3d68af29de8"
+  revision 1
+  head "https://github.com/aws/aws-ec2-instance-connect-cli.git", :branch => "master"
+
+  depends_on "python"
+
+  resource "botocore" do
+    url "https://files.pythonhosted.org/packages/56/28/9ef10dca4aa0d48802765447e3f6371dc218567b0f3038bdb7664ae4f0e0/botocore-1.12.253.tar.gz"
```

Comment:
You probably can bump this to the latest, `v1.13.3`?

Reply:
Adjusted